browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -11,9 +11,12 @@
 
 #include "base/bind.h"
 #include "brave/app/brave_command_ids.h"
+#include "brave/browser/brave_wallet/brave_wallet_context_utils.h"
```

Comment:
Note: I haven't reverted the change to this file as we might be adding the share button here.